python/ray/rllib/agents/ddpg/ddpg_policy_graph.py

```diff
@@ -355,7 +447,8 @@ def compute_td_error(self, obs_t, act_t, rew_t, obs_tp1, done_mask,
         return td_err
 
     def reset_noise(self, sess):
-        sess.run(self.reset_noise_op)
+        if not self.config["use_gaussian_noise"]:
```

Comment:
This is no longer a valid flag right? Maybe we can always run this since resetting noise is harmless in the other case.

Reply:
my fault....replaced.